parser-typechecker/src/Unison/Runtime/Rt1IO.hs

```diff
@@ -489,6 +489,15 @@ toTermOrError ppe r = case r of
       P.wrap "This happens when calling a function that doesn't handle all possible inputs.",
       "", sorryMsg
       ]
+  Right (RT.RError msg) ->
+    pure . Left . P.callout icon . P.lines $ [
+      P.wrap ("I've encountered a call to a" <> P.red "error function"
+              <> "with the following message:"), "",
+      P.indentN 2 $ P.wrapString msg,
+      "",
+      P.wrap "This happens when calling one of the builtin error functions like `bug` or `todo`.",
+      "", sorryMsg
+      ]
```

Comment:
With the change I suggested above, you can now give a different error depending on whether the crash is due to a `bug` or `todo` call. You can decompile the `Value` using `IR.decompile`  and  you can use the `TermPrinter` to render the term (not `show`): 

```Haskell
pretty :: Var v => PrettyPrintEnv -> AnnotatedTerm v a -> Pretty ColorText
```